README.md

```diff
@@ -54,4 +54,17 @@ Payload fields:
 - `timestamp`: The timestamp when the issue was detected.
 - `labels`: Metadata or tags that provide additional context (e.g., app name, Kubernetes namespace, and cluster information).
 - `entityId`: The unique identifier for the entity involved.
-- `objectId`: A unique identifier for the specific object associated with this event, in this case it's the defectID.
\ No newline at end of file
+- `objectId`: A unique identifier for the specific object associated with this event, in this case it's the defectID.
+
+## Filtering Notifications
+A very simple filter configuration has been provided to filter notifications based on `problemType` and `entityType`. This can be set by customizing the `filterconfig` map as seen below:
+
+```yaml
+filterConfig:
```

Comment:
so the question is if you specify two types of entities (like container and application), does malfunction apply to both or only certain combination?

Reply:
To Both so for example if the config was `entityTypes`: [Container, Application] and `problemTypes`: [Malfunction] then the only allowed notifications would be `(Container, Malfunction)` and `(Application, Malfunction)`

Reply:
if that means every possible combination of entities and problems listed, that may not be specific enough for the user

Reply:
I was also thinking about this.  I think a list of pairs Pod - Malfunction, Container - CPUCongestion ... is more useful to the user.  That being said, we can easily adjust this.  This is intended to give the user a starting point for filtering.

Reply:
Addressed in the latest commit!